### PR TITLE
Add readme.md

### PR DESCRIPTION
**Pull request by peteronyam**
Add readme
Description:

This pull request updates the Readme file to enhance the project's readability and provide more comprehensive information about the project's purpose and goals. The changes aim to help new contributors understand the project and how to contribute to it by adding installation instructions, troubleshooting steps, and a step-by-step guide for building and running the project.

Changes Made:

The following changes were made to the Readme file:

Updated the project description to provide a clear and concise explanation of the project's purpose and goals.
Added installation instructions and troubleshooting steps to assist new contributors in getting started with the project quickly.
Included a step-by-step guide for building and running the project, along with guidelines for contributing to the project.
Updated the list of dependencies and requirements to ensure that contributors have all the necessary tools and resources to work on the project.
Added a section on running tests and troubleshooting common issues that may arise during the development process.
Benefits:

The changes to the Readme file offer several benefits to the project, including:

Improved accessibility and ease of use for new contributors, decreasing the learning curve and assisting them in getting up to speed quickly.
Increased clarity and transparency regarding the project's purpose and goals, making it easier for potential users to comprehend the project's value and how it can benefit them.
Improved overall project documentation, making it simpler to maintain and update as the project evolves over time.
Additional Notes:

The pull request has undergone several reviews, and all necessary modifications have been made. The new information has been tested and validated to ensure its accuracy and completeness.

We kindly request that the project maintainers review and merge this pull request as soon as possible since it is a vital update to the project's documentation, assisting new contributors in getting started quickly.

### CODE REVIEW

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @peteronyam, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

- [ ] You did a great job using the Microverse README template! 💯 However, please make sure that you remove the guiding comments from all sections! as they serve as just boilerplate text so you have an idea of what to include in each section! 🤓 (Image only shows 3).

imageedit_4_9524725346


- [ ] Please consider including some content in your "Getting started" section, this is one of the most important ones for collaboration! Here you guide other devs on how they can get a local copy of the project up and running. 🤓 🔧 

image

- [ ] Please consider editing your PR description including detailed information about what you're adding to the project in this Pull request. ⚠️ **Also, for future projects, please avoid naming your branch "feature", instead, use the actual name of the feature you're implementing, in this case, it could be "hello-microverse". PLEASE DO NOT ATTEMPT TO CHANGE IT HERE SINCE IT CAN CAUSE SOME UNEXPECTED ISSUES WITH YOUR COMMIT HISTORY.** ⚠️ 🤓 

imageedit_7_8644075940

- [ ] According to the requirements, you should have a `.gitignore` file with the `node_modules` in it:

image

- [ ] Also, there's a task that requires you to create a `test.md` file and make sure it's ignored by git. However, I'm unable to see this. Please consider doing it! 🤓 🔧 

image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -117,9 +112,7 @@ After you're finished please remove all the comments and instructions!
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
```
Comment (Kamba56) [inline README.md:89]: - [ ] As requested by the previous reviewer, kindly fill in all the necessary sub sections under the getting started section. The prerequisites are those things a user will reuwire before they can clone your project. You add a comment to clone under the `clone this repo with:` etc.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @peteronyam, 

Great work on making the changes requested by a previous reviewer 👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed!👏👏👏

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by peteronyam** [comment]
Please, my coding partner submitted this readme template just the way it is
yesterday without making changes to the comments sections at all he only
added his usernames where we were required to add our social profile and
that’s it, why am I passing through this, please
Plus the fact that I have only one time left to submit this project for
review

**Comment by peteronyam** [comment]
Can you please, check that now
I have made the changes as required
Plus I only have one time left to submit this for review please

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @peteronyam 👋, 

Great work on making the changes requested by a previous reviewer 👏 

## Highlights
- All linters checks passed ✔️ 
- `README.md` file is professional ✔️ 
- `node_modules` and the `test.md` file were git ignored ✔️ 

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Required Changes ♻️
- [ ]  The first reviewer requested that you edit the pull request summary. Kindly edit your pull request summary and describe/summarize what you did in this project. Pull request summaries help reviewers in the professional world have an idea of what changes were introduced in a feature branch

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gambit142** [comment]
> Can you please, check that now I have made the changes as required Plus I only have one time left to submit this for review please

Great work @peteronyam, the only thing you need to do is edit your pull request summary. 👍

**Comment by peteronyam** [comment]
> > Can you please, check that now I have made the changes as required Plus I only have one time left to submit this for review please
> 
> 
> 
> Great work @peteronyam, the only thing you need to do is edit your pull request summary. 👍 

I have added the message, please help review it, you know that was my last review request, please I have done everything.

**Comment by Gambit142** [comment]
> > > Can you please, check that now I have made the changes as required Plus I only have one time left to submit this for review please
> > 
> > 
> > Great work @peteronyam, the only thing you need to do is edit your pull request summary. 👍
> 
> I have added the message, please help review it, you know that was my last review request, please I have done everything.

Hi @peteronyam, you are good to go. You have edited the pull request summary. Great work 👏

**Comment by peteronyam** [comment]
> > > > Can you please, check that now I have made the changes as required Plus I only have one time left to submit this for review please
> 
> > > 
> 
> > > 
> 
> > > Great work @peteronyam, the only thing you need to do is edit your pull request summary. 👍
> 
> > 
> 
> > I have added the message, please help review it, you know that was my last review request, please I have done everything.
> 
> 
> 
> Hi @peteronyam, you are good to go. You have edited the pull request summary. Great work 👏 

Why is it still showing "pending review" ?

**Review by souad988 — APPROVED** [review]
Hi @peteronyam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
- Try to use a short PR summary that describes the work done in the feature. 
   
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by peteronyam** [comment]
Merging pull requests from the feature branch to the main branch

**Pull request merged**